exchanges/btse/btse_wrapper.go

```diff
@@ -1129,16 +1129,17 @@ func (e *Exchange) GetFuturesContractDetails(ctx context.Context, item asset.Ite
 		}
 
 		c := futures.Contract{
-			Exchange:             e.Name,
-			Name:                 cp,
-			Underlying:           currency.NewPair(currency.NewCode(marketSummary[i].Base), currency.NewCode(marketSummary[i].Quote)),
-			Asset:                item,
-			SettlementCurrencies: settlementCurrencies,
-			StartDate:            startTime,
-			EndDate:              endTime,
-			SettlementType:       contractSettlementType,
-			IsActive:             marketSummary[i].Active,
-			Type:                 ct,
+			Exchange:                       e.Name,
+			Name:                           cp,
+			Underlying:                     currency.NewPair(currency.NewCode(marketSummary[i].Base), currency.NewCode(marketSummary[i].Quote)),
```

Comment:
suggestion/oos: Base and Quote could be currency.Code types.

Reply:
Main reason against is this line:
```
cp, err := currency.NewPairFromString(marketSummary[i].Base, marketSummary[i].Symbol[len(marketSummary[i].Base):])
```
Feels more convoluted with
```
cp := currency.NewPair(marketSummary[i].Base, currency.NewCode(marketSummary[i].Symbol[len(marketSummary[i].Base.String()):]))
```

There may be a better way once I sleep more

Reply:
I've asked for that line to be changed anyway.
I'd be supportive of using currency.Code for both.